tests/wpt/web-platform-tests/html/semantics/forms/textfieldselection/selection-value-interactions.html

```diff
@@ -97,22 +97,31 @@
     var el = document.createElement(tag);
     document.body.appendChild(el);
     this.add_cleanup(() => el.remove());
-    el.value = "";
-    assert_equals(el.selectionStart, el.value.length,
-                 "element.selectionStart should be value.length");
-    assert_equals(el.selectionEnd, el.value.length,
-                  "element.selectionEnd should be value.length");
-    el.value = "foo";
-    assert_equals(el.selectionStart, el.value.length,
-                 "element.selectionStart should be value.length");
-    assert_equals(el.selectionEnd, el.value.length,
-                  "element.selectionEnd should be value.length");
-    el.value = "foobar";
-    assert_equals(el.selectionStart, el.value.length,
-                 "element.selectionStart should be value.length");
-    assert_equals(el.selectionEnd, el.value.length,
-                  "element.selectionEnd should be value.length");
-  }, `selection is always collapsed to the end after setting values on ${tag}`);
+
+    for (let val of ["", "foo", "foobar"]) {
```

Comment:
I think upstream W3C does not allow ES6 syntax, so we'll have to use `var` instead of `let` here.

Reply:
Are you sure about this? There is already loads of ES6 syntax in the existing tests (try `ag "=>" tests/wpt/web-platform-tests`). I used the backticks as I'd already seen them in another test, and then assumed that `let` would also be fine.

Reply:
ES6 is fine.

Reply:
Nevermind me, false alarm. WPT does support ES6 syntax.

Reply:
Ok great, so I think this can be merged? Was there something else?